src/Geopilot.Api/ContextExtensions.cs

```diff
@@ -11,7 +11,7 @@ namespace Geopilot.Api;
 
 internal static class ContextExtensions
 {
-    internal const string UserIdClaim = "oid";
+    internal const string UserIdClaim = "sub";
```

Comment:
Q: Nur für mein Verständnis 😅 Führt dieser Change dazu, dass bei bestehenden Umgebungen der `AuthIdentifier` in der DB aktualisiert werden muss, oder bleibt der gleich?

Reply:
@flenny Leider ja. Hier hatten wir einen Micrsosoft speziefischen [Doku](https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#payload-claims) verwendet.